src/maidsafe/vault/operation_visitors.h

```diff
@@ -370,6 +370,26 @@ class DataManagerAccountQueryVisitor : public boost::static_visitor<> {
 };
 
 template <typename ServiceHandlerType>
+class MpidManagerAccountQueryVisitor : public boost::static_visitor<> {
+ public:
+  MpidManagerAccountQueryVisitor(ServiceHandlerType* service,
+                                 const NodeId& sender_node_id,
```

Comment:
Again, we can take these two `NodeId`s by value and move them into their targets.